xrtpy/response/channel.py

```diff
@@ -432,7 +432,7 @@ def number_of_wavelengths(self):
 
     @property
     def observatory(self) -> str:
-        """Spacecraft: Hinode satellite."""
+        """The spacecraft name."""
```

Comment:
It appears that starting a docstring (or line?) with a word before a colon makes it so that the word is intended to be a reference object in a nitpicky doc build. Not sure why, but these slight edits should make the warnings go away.